lib/test/tasks/run_file_size_checks.rb

```diff
@@ -10,7 +10,7 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
     'check_ins*.css' => (10..15),
     'check_ins*.js' => (197..207),
     'check_ins_index*.js' => (5..15),
-    'ci_step_gantt_charts*.js' => (830..840),
+    'ci_step_gantt_charts*.js' => (806..816),
```

Comment:
It's always nice to see a decrease (though, in absolute terms, this file is still pretty huge).

I think that this might possibly be because there is no longer duplication in one of the vega packages, but I'm not very sure about that at all.